.gitignore

```diff
@@ -164,4 +164,6 @@ internal/*
 *_pitch.npy
 *_phoneme.npy
 wandb
-depot/*
\ No newline at end of file
+depot/*
+bin
+pyvenv.cfg
```

Comment:
what are these ?

Reply:
these are files that got created when creating virtual env.